pkg/policy/filter.go

```diff
@@ -0,0 +1,52 @@
+package policy
```

Comment:
erm

Reply:
?

Reply:
A subtle way of saying unrelated change?